File: app/src/main/java/com/antyzero/smoksmog/fabric/FabricModule.java
package com.antyzero.smoksmog.fabric;


import com.crashlytics.android.answers.Answers;
import com.crashlytics.android.core.CrashlyticsCore;

import javax.inject.Singleton;

import dagger.Module;
import dagger.Provides;

@Singleton
@Module
public class FabricModule {

    @Provides
    @Singleton
    Answers provideAnswers() {
        return Answers.getInstance();
    }

    @Provides
    @Singleton
    CrashlyticsCore provideCrashlyticsCore() {
        return CrashlyticsCore.getInstance();
    }
}

File: app/src/main/kotlin/com/antyzero/smoksmog/fabric/FabricModule.kt
package com.antyzero.smoksmog.fabric


import com.crashlytics.android.answers.Answers
import com.crashlytics.android.core.CrashlyticsCore
import dagger.Module
import dagger.Provides
import javax.inject.Singleton

@Singleton
@Module
class FabricModule {

    @Provides
    @Singleton
    internal fun provideAnswers(): Answers {
        return Answers.getInstance()
    }

    @Provides
    @Singleton
    internal fun provideCrashlyticsCore(): CrashlyticsCore {
        return CrashlyticsCore.getInstance()
    }
}
